Week2/homework/js-exercises/4-reading-list.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+//Creating an array of objects
+
+const bookList = [
+    {
+     title: "One Hundred Years of Solitude",
+     author: "Gabriel Garcia Marquez",
+     alreadyRead: true
+    },
+    {
+    title:"The Blind Owl",
+    author:"Sadegh Hedayat",
+    alreadyRead: false
+    },
+    {
+    title: "Ince Memed",
+    author: "Yaşar Kemal",
+    alreadyRead: true
+    }
+]
+
+// Loop through the array
+
+let i = 0;
+
+const bookKeys = Object.keys(bookList[i]);
+for(i = 0; i < bookList.length; i++){
+    console.log(bookList[i]);
+}
+
+console.log("<---->")
+
+//Title and author
+for(i = 0; i < bookList.length; i++){
+console.log(bookList[i][bookKeys[0]] + " by " + bookList[i][bookKeys[1]]);
```

Comment:
Although this works, it is much easier to read code that looks like this:
```
for(i = 0; i < bookList.length; i++) {
  console.log(bookList[i].title + " by " + bookList[i].author);
}
```

In addition, if you decide to add more keys to the `bookList` object, the ordering of keys might change, so it is not guaranteed that 0 will be title, 1 will be author, etc.